cpu/qn908x/include/cpu_conf.h

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright (C) 2020 iosabi
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_qn908x
+ * @{
+ *
+ * @file
+ * @brief       Implementation specific CPU configuration options
+ *
+ * @author      iosabi <iosabi@protonmail.com>
+ */
+
+#ifndef CPU_CONF_H
+#define CPU_CONF_H
+
+#include "cpu_conf_common.h"
+
+#include "vendor/QN908XC.h"
+#include "vendor/QN908XC_features.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name   ARM Cortex-M specific CPU configuration
+ * @{
+ */
+#define CPU_DEFAULT_IRQ_PRIO    (1U)
+/**
+ * NUMBER_OF_INT_VECTORS in the QN908XC.h is defined as including the standard
+ * ARM interrupt vectors and headers, however CPU_IRQ_NUMOF does not include
+ * the first 15 interrupt values and the stack pointer.
+ */
+#define CPU_IRQ_NUMOF           (NUMBER_OF_INT_VECTORS - 16)
+/**
+ * The flash is aliased at several addresses in the memory range. In particular,
+ * address 0 can be mapped to RAM or flash, so it is possible to run from
+ * address 0 from flash, or even turn off the flash altogether and run from RAM
+ * to save power. This setting uses the ROM_START_ADDR value set in the
+ * Makefile.
+ */
+#define CPU_FLASH_BASE          (QN908X_ROM_START_ADDR)
+/** @} */
```

Comment:
You can also set

```C
#define CPU_HAS_BITBAND 1
```

here

![image](https://user-images.githubusercontent.com/1301112/100749550-16530900-33e5-11eb-915d-7609e1ac902a.png)

Reply:
Oh thanks, I missed that part.